libs/klv/src/main/java/org/codice/alliance/libs/klv/AttributeNameConstants.java

```diff
@@ -57,11 +65,11 @@
 
     public static final String OBJECT_COUNTRY_CODES = "object-country-codes";
 
-    public static final String CHECKSUM = "checksum";
+    public static final String CHECKSUM = "klv-packet-checksum";
 
     public static final String IMAGE_COORDINATE_SYSTEM = "image-coordinate-system";
```

Comment:
If this is the coordinate reference system, it can be mapped to location.crs-code